templates/default/fulldoc/html/js/full_list.js

```diff
@@ -62,8 +62,25 @@ function enableToggles() {
     evt.stopPropagation();
     evt.preventDefault();
     $(this).parent().parent().toggleClass('collapsed');
+    $(this).attr('aria-expanded', function (i, attr) {
+        return attr == 'true' ? 'false' : 'true'
+    });
     highlight();
   });
+
+  // navigation of nested classes using keyboard
+  $('#full_list a.toggle').on('keypress',function(evt) {
```

Comment:
Part of this change allows you to tab navigate to the arrow/carrot in the full list menu, then you can press enter to expand the section without visiting a new page.